ceph_iscsi_config/gateway_setting.py

```diff
@@ -184,6 +184,7 @@ def normalize(self, raw_val):
     "minimum_gateways": IntSetting("minimum_gateways", 1, 9999, 2),
     "ceph_config_dir": StrSetting("ceph_config_dir", '/etc/ceph'),
     "priv_key": StrSetting("priv_key", 'iscsi-gateway.key'),
+    "gateway_conf": StrSetting("gateway_conf", 'iscsi-gateway.conf'),
```

Comment:
Nit: perhaps keep `priv_key` and `pub_key` next to each other since they are linked.

Reply:
> Nit: perhaps keep `priv_key` and `pub_key` next to each other since they are linked.

Ok,i fix it.thank you .